WalletWasabi.Gui/Tabs/WalletManager/RecoverWallets/RecoverWalletViewModel.cs

```diff
@@ -49,6 +49,9 @@ public RecoverWalletViewModel(WalletManagerViewModel owner) : base("Recover Wall
 
 				string walletFilePath = WalletManager.WalletDirectories.GetWalletFilePaths(WalletName).walletFilePath;
 
+				int.TryParse(MinGapLimit, out int minGapLimit);
```

Comment:
Do not ignore the result. It lets you know if the value is an int or not.

Reply:
But I don't want to only know if it's an int or not, I also want to know if it is between 21 and 1 million.

Reply:
if it is not an `int` then you should not continue verifying anything. Never ignore error.

Reply:
btw, 1 millon is crazy. It is not only that it can take a long time in many computers but also this makes wasabi to download all the blocks because the filters will almost always match.

Reply:
> if it is not an int then you should not continue verifying anything. Never ignore error.

Fixed.

> btw, 1 millon is crazy.

What number should we lower it to? 10_000?